query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/fields/LdapAttributeName.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.admin.ldap.fields;
+
+import static org.codice.ddf.admin.ldap.commons.LdapMessages.invalidUserAttribute;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import org.codice.ddf.admin.api.report.ErrorMessage;
+import org.codice.ddf.admin.common.fields.base.scalar.StringField;
+
+public class LdapAttributeName extends StringField {
+
+    public static final String DEFAULT_FIELD_NAME = "attriName";
+
+    public static final String FIELD_TYPE_NAME = "LdapAttributeName";
+
+    public static final String DESCRIPTION =
+            "The short descriptive name of an LDAP attribute as defined in RFC2251.";
```

Comment:
❗️ As @blen-desta correctly informed me, the RFC referenced here should be RFC4512.